### PR TITLE
Add a first shot at visualize.py with the PCA viz

### PR DESCRIPTION
**Pull request by yacineMahdid**

### CODE REVIEW

File: src/visualization/visualize.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
+import pandas as pd
+import numpy as np
+from sklearn.preprocessing import Normalizer
+
+DATA_SOURCE = "/home/yacine/Documents/Kaggle/heart_disease_eda/data/raw/heart_kaggle.csv"
+FIGURE_SOURCE = "/home/yacine/Documents/Kaggle/heart_disease_eda/reports/figures/"
+
+df = pd.read_csv(DATA_SOURCE)
```
Comment (yacineMahdid) [inline src/visualization/visualize.py:8]: thats not a good normalization....
